_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,92 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Basic Setup
+---------------------------- */
+html {
+    height: 102%;
+}
+
+body {
+    font-size: $font-size;
+    min-width: $fg-min-width;
+    background: $gray-lightest;
+    color: $gray-darker;
+}
+
+body,
+input,
+select,
+textarea,
+button {
+    font-family: $font-sans;
+}
+
+a {
+    @include transition(color $timing-fastest ease-in-out 0s);
+    text-decoration: none;
+    color: $blue;
+
+    &:hover,
+    &:focus {
+        text-decoration: underline;
+        color: $orange-dark;
+    }
+}
+
+h1,
+h2,
+h3 {
+    @include page-title('sans');
+}
+
+h4,
+h5,
+h6 {
+    @include page-title('serif');
+}
+
+p {
+    @include font-size();
+    @include line-height();
+}
+
+ul,
+ol {
+
+    li {
+
+    }
+}
+
+figure {
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+
+    figcaption {
+        @include line-height();
+        font-family: $font-serif;
+        font-style: italic;
+    }
+}
+
+blockquote {
+
+}
+
+.icon {
+    @extend %icon-font;
+}
+
+[class^="wrapper-"],
+[class*=" wrapper-"] {
```

Comment:
Hmm, why don't we use the more general selector of ``[class*="wrapper"]`` attribute selector then to catch both?